CHANGELOG.md

```diff
@@ -45,16 +45,16 @@ Classify the change according to the following categories:
 ## v2.12.0
 ### Minor Updates
 ##### Added
```

Comment:
@Bill-Becker  is this intentional to replace "job\" with "reopjl/" throughout the Changeling?

Reply:
Good question, because I could see it going either way, but I feel like if we are renaming the app then we'd want to change it in the changelog because people reading the changelog would know what app we're referring to. Did you see someplace where it didn't look right? The only weird place where `job` is still relevant is the `job/<run_uuid>/results` URL which is where you grab the results, so I think as long as I didn't replace that we should be good. Also, the `test_job_endpoint.jl` is still properly named because it's referring to the `reoptjl` app's `/job/results` endpoint